src/main/java/club/sk1er/patcher/mixins/features/GuiIngameForgeMixin_TitleRendering.java

```diff
@@ -0,0 +1,53 @@
+package club.sk1er.patcher.mixins.features;
+
+import club.sk1er.patcher.config.PatcherConfig;
+import gg.essential.universal.UResolution;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.FontRenderer;
+import net.minecraft.client.gui.GuiIngame;
+import net.minecraftforge.client.GuiIngameForge;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.injection.*;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+import org.spongepowered.asm.mixin.injection.invoke.arg.Args;
+
+@Mixin(GuiIngameForge.class)
+public class GuiIngameForgeMixin_TitleRendering extends GuiIngame {
+
+    @Shadow
+    private FontRenderer fontrenderer;
+
+    public GuiIngameForgeMixin_TitleRendering(Minecraft mc) {
+        super(mc);
+    }
+
+    @Inject(method = "renderTitle", at = @At("HEAD"), cancellable = true, remap = false)
+    private void patcher$cancelTitleRender(CallbackInfo ci) {
+        if (PatcherConfig.disableTitles) ci.cancel();
+    }
+
+    @ModifyArgs(method = "renderTitle", at = @At(value = "INVOKE", target = "Lnet/minecraft/client/renderer/GlStateManager;scale(FFF)V"))
```

Comment:
This should just be done by scaling yourself in an inject. Changing the original code is unnecessarily brittle and less readable. Same goes for everything below this.

Reply:
Done